app/src/main/java/com/conupods/OutdoorMaps/Services/LocationService.java

```diff
@@ -0,0 +1,120 @@
+package com.conupods.OutdoorMaps.Services;
+
+import android.Manifest;
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.os.Build;
+import android.os.IBinder;
+import android.os.Looper;
+import android.util.Log;
+
+import com.conupods.OutdoorMaps.CameraController;
+
+import androidx.activity.ComponentActivity;
+import com.conupods.OutdoorMaps.View.MapsActivity;
+import com.google.android.gms.common.api.ResolvableApiException;
+import com.google.android.gms.location.FusedLocationProviderClient;
+import com.google.android.gms.location.LocationCallback;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationResult;
+import com.google.android.gms.location.LocationServices;
+import com.google.android.gms.location.LocationSettingsRequest;
+import com.google.android.gms.location.LocationSettingsResponse;
+import com.google.android.gms.location.SettingsClient;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.app.ActivityCompat;
+import androidx.core.app.NotificationCompat;
+
+public class LocationService extends Service {
+
+    CameraController mCameraController;
+    private FusedLocationProviderClient mFusedLocationClient;
+
+
+    private static final int RESOLVABLE_API_ERROR_REQUEST_CODE = 51;
+    private static final String TAG = "LocationService";
```

Comment:
all tags are in caps with underscores, this is for consistent error reporting with my code! I think I'm the only one who had to log errors with tags if I'm not mistaken.

Reply:
I use TAG to keep up to date with what is being done in the class (important actions). Do you want it to be deleted?

Reply:
Changed to follow the convention